frontend/src/scenes/data-warehouse/new/NewSourceWizard.tsx

```diff
@@ -17,14 +16,43 @@ import { dataWarehouseTableLogic } from './dataWarehouseTableLogic'
 import { sourceWizardLogic } from './sourceWizardLogic'
 
 export const scene: SceneExport = {
-    component: NewSourceWizard,
+    component: NewSourceWizardScene,
```

Comment:
Wasn't changed in this PR so can't comment on the exact line but needs more space above the actions.

![Screenshot 2024-07-08 at 5 29 57 PM](https://github.com/PostHog/posthog/assets/6392501/205d6eef-780c-409e-87fa-7356d711a5f5)